cfgov/cal/views.py

```diff
@@ -63,11 +120,13 @@ def display(request, pdf=False):
     """
     display (potentially filtered) html view of the calendar
     """
+
     template_name='about-us/the-bureau/leadership-calendar/index.html'
-    form = CalendarFilterForm(request.GET)
+    form = CalendarPDFForm(request.GET) if pdf else CalendarFilterForm(request.GET)
     context = {'form': form}
+    form_is_vaild = form.is_valid()
 
-    if form.is_valid():
+    if form_is_vaild:
         cal_q = Q(('active', True))
         if form.cleaned_data.get('filter_calendar', None):
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/pull/2263/commits/8b1241190c12d6ca8aef3094ad43dcefe3459d7e.